scipioninstaller/installer.py

```diff
@@ -62,7 +62,7 @@ def getCondaCmd(scipionEnv, noAsk, create):
     cmd = cmdfy(getCondaInitCmd())
     if create:
         silentMode = "-y" if noAsk else ""
-        cmd += cmdfy("%s create %s -n %s python=3.8" % (CONDA, silentMode, scipionEnv))
+        cmd += cmdfy("%s create %s -n %s python=3" % (CONDA, silentMode, scipionEnv))
```

Comment:
Thanks @azazellochg for suggesting this change. I've got bad experiences using the latest version of whatever "package".

In particular with python 3.9 (latest at the time I was testing this installer), some of the dependencies weren't available as "precompiled packages", therefore the "compiling from sources" was triggered. At least, matplotlib was involved and failed to install. I found so many links suggesting to give up on compiling matplotlib from sources. Maybe it would be the same for numpy, scipy, others?. 

Maybe, currently 3.9 is fine, or maybe 3.10 (would need test) but, I personally prefer not to leave the version open to the latest but a specific one.

Maybe, what we can do is to offer a parameter to "optionally", choose the python version.

Thought?

Reply:
Hi Pablo, sure we can fix it to one version (3.10?). Scipion deps seem to be installing okay. The only problem is mpi4py which usually never works, since it requires mpicc inside conda. I have not tested xmipp deps since I dont know where they are listed.